src/tests/unit/tests/DetailsView/components/assessment-view-update-handler.test.ts

```diff
@@ -158,6 +161,48 @@ describe('AssessmentViewTest', () => {
 
             detailsViewActionMessageCreatorMock.verifyAll();
         });
+
+        test('do not enable for getting started view', () => {
```

Comment:
Should we add the following to make the test name clearer?
```suggestion
        test('do not enable visual helper for getting started view', () => {
```

Reply:
I like that suggestion @lisli1 !